### PR TITLE
Portfolio: setup and mobile first

### PR DESCRIPTION
**Pull request by Tshepo45**
In this PR:
-Modified linters script
-Modified Readme file
-Modified .html and .css files

Thank you for reviewing my PR

### CODE REVIEW

**Review by levy002 — COMMENTED** [review]
## Hello @Tshepo45 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :x: :x: 

You have submitted a project, that doesn't have a professional PR title and PR summary.

- [ ] Kindly make sure to update your PR title and PR description by proving a well-detailed summary and title which makes your PR clean and professional and even self-explanatory. :wink: 

   - Example of good PR Summary
image

  ### HOW TO EDIT PR DESCRIPTION
   chrome-capture-2023-3-27

 - Example of a good PR title
    _**Portfolio: setup and mobile first**_

   ### HOW TO EDIT PR TITLE
    chrome-capture-2023-3-27 (1)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: styles.css [hunk 1]
```diff
@@ -55,11 +55,6 @@ nav a {
   padding-right: 10rem;
 }
```
Comment (mukhammadrizooff) [inline styles.css:56]: - [ ] [REQUIRED] Good job so far! Please remove necessary `space` from right side of your web page and implement `border-radius` with required value to battom part of the section 🙏🏼 

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @Tshepo45, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Correct GitHub Flow ✔️ 
- Good implementations ✔️ 

## Required Changes ♻️
- [ ] [REQUIRED] Good job so far! It would be more semantic if you implement your `index.html` file like that:
```html
<body>
    <Header class="Toolbar">
        <nav>
            <ul>
              <li><a href="#">Tshepo</a> </li>
              <li><a href="#"><img src="image/Union.png" alt="Union"></a></li>
            </ul> 
        </nav>
    </Header>
    <section class="container1">
        <div class="P-Text">
            <h1>I’m Tshepo <br> Glad to see <br> you!</h1>
            <p>I’m a software developer! I can help you <br> build a product , feature or website Look <br> through some of my work and <br> experience! If you like what you see and <br> have a project you need coded, don’t <br> hestiate to contact me.</p>
            <a href="#" class="Connect">LET’S CONNECT</a>
            
            <ul class="Social">
                <li><a href="#"><img src="image/Vector5.png" alt=""> </li>
                <li><a href="#"><img src="image/Linkedin icon (1).png" alt=""></a></li>
                <li><a href="#"><img src="image/Vector1.png" alt=""></a></li>
                <li><a href="#"><img src="image/Vector2.png" alt=""></a></li>
                <li><a href="#"><img src="image/Vector3.png" alt=""> </a></li>
            
            </ul>
        </div>
    </section>
</body>
```

image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -18,7 +18,7 @@
                </nav>
             </Header>
         <div class="P-Text">
```
Comment (victorokoroji) [inline index.html:20]: - [ ] While trying to remove the extra spacing from the right of your web page, you ended up adding the spacing to the left. I think you can make proper use of Flexbox to fix this issue and set the entire view width to 100%

image

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @Tshepo45 , 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by roshan-bajgain — APPROVED** [review]
Hi @Tshepo45, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**